gnuradio-runtime/lib/pmt/pmt.cc

```diff
@@ -137,7 +137,7 @@ bool to_bool(pmt_t val)
 
 static const unsigned int get_symbol_hash_table_size()
 {
-    static const unsigned int SYMBOL_HASH_TABLE_SIZE = 701;
```

Comment:
does anyone know what the reasoning for 701 originally was.  seems like an oddly specific table size

Reply:
There's speculation on that question here: https://github.com/gnuradio/gnuradio/pull/1448#issuecomment-326732363

Reply:
I am currently re-running the distribution tests with random words, and the word list here https://github.com/dwyl/english-words and will update with results.

Reply:
Results are aligned with @marcusmueller observations, and std::hash still does a much better job:

![image](https://user-images.githubusercontent.com/36242998/96047096-d1bddf00-0e31-11eb-884b-566077f6c594.png)

466k words were from the list above, the rest are random strings.

Reply:
(Removed my other account's comments in favor of one consolidated comment)

I am re-ran the distribution tests with the word list here https://github.com/dwyl/english-words and a few random strings. The `hash_string()` results are well aligned with what @marcusmueller observed previously for the custom GR hash and a 2^n table size, but std::hash returns a very reasonable distribution:

![image](https://user-images.githubusercontent.com/7121282/96052815-2150da80-0e33-11eb-86c3-19cd10fbd6e8.png)

It's probably more useful to view the hash value distribution directly (histogram of above):

![image](https://user-images.githubusercontent.com/7121282/96052542-a2f43880-0e32-11eb-9e8d-f94bc693db53.png)

Overall this looks as one would expect.